containers/README.md

```diff
@@ -8,17 +8,34 @@ This folder holds source code and Docker configurations for the container images
 
 This section describes the process for building and pushing container images to Artifact Registry, where they will later be used to deploy to a container runtime environment like GKE.
```

Comment:
Can you update this text to be more descriptive of what we're doing now, ie: mention Cloud Deploy, etc, etc.... A touch more detail here would be good.